Makefile

```diff
@@ -30,7 +30,11 @@ check-virtualenv:
 bootstrap: check-virtualenv install-deps
 
 install-deps:
-	pip install 'setuptools>=20.8.1'
+	if [[ "$(python -V)" == 'Python 3.3.*' ]]; then \
+		setuptools_min_version="18"; \
+	else \
+		setuptools_min_version="20.8.1"; \
```

Comment:
hm, I thought the reason to we pinned to 20.8 was that earlier versions didn't work with some syntax we had.

Reply:
IDK I just know it doesn't support Python 3.3.

Reply:
I think this is what introduced the bug in Travis and broke 3.3 builds: https://github.com/pypa/setuptools/commit/114bb58167d4f6462e0b7d69a46233b796840ba8.

Reply:
Nvm that is a setuptools Travis file, not Travis's copy of setuptools. Here is a similar issue: https://github.com/aws/base64io-python/issues/4.

Reply:
And this: https://github.com/travis-ci/travis-ci/issues/9002.

Reply:
OK I just figured it out... I believe that setuptools dropped support for 3.3 in this diff: https://github.com/pypa/setuptools/commit/7392f01ffced3acfdef25b0b2d55cefdc6ee468a. That is when `pip install "setuptools>=20"` stopped working. Release 40 is `>=20`, yet does not support Python 3.3. I might even change the constraint in Python 3.3 to reflect this issue.